plugins/nodes/src/js/components/NodesTable.tsx

```diff
@@ -159,124 +166,135 @@ export default class NodesTable extends React.Component<
-          />
-
-          <Column
-            header={
-              <SortableHeaderCell
-                columnContent={<Trans render="span">Region</Trans>}
-                sortHandler={this.handleSortClick.bind(null, "region")}
-                sortDirection={sortColumn === "region" ? sortDirection : null}
-              />
-            }
-            cellRenderer={this.regionRenderer}
-          />
-
-          <Column
-            header={
-              <SortableHeaderCell
-                columnContent={<Trans render="span">Zone</Trans>}
-                sortHandler={this.handleSortClick.bind(null, "zone")}
-                sortDirection={sortColumn === "zone" ? sortDirection : null}
-              />
-            }
-            cellRenderer={zoneRenderer}
-          />
-
-          <Column
-            header={
-              <SortableHeaderCell
-                columnContent={<Trans render="span">Tasks</Trans>}
-                sortHandler={this.handleSortClick.bind(null, "tasks")}
-                sortDirection={sortColumn === "tasks" ? sortDirection : null}
-                textAlign="right"
-              />
-            }
-            cellRenderer={tasksRenderer}
-          />
-
-          <Column
-            header={
-              <SortableHeaderCell
-                columnContent={<Trans render="span">CPU</Trans>}
-                sortHandler={this.handleSortClick.bind(null, "cpu")}
-                sortDirection={sortColumn === "cpu" ? sortDirection : null}
-              />
-            }
-            cellRenderer={cpuRenderer}
-          />
-
-          <Column
-            header={
-              <SortableHeaderCell
-                columnContent={<Trans render="span">Mem</Trans>}
-                sortHandler={this.handleSortClick.bind(null, "mem")}
-                sortDirection={sortColumn === "mem" ? sortDirection : null}
-              />
-            }
-            cellRenderer={memRenderer}
-          />
-
-          <Column
-            header={
-              <SortableHeaderCell
-                columnContent={<Trans render="span">Disk</Trans>}
-                sortHandler={this.handleSortClick.bind(null, "disk")}
-                sortDirection={sortColumn === "disk" ? sortDirection : null}
-              />
-            }
-            cellRenderer={diskRenderer}
-          />
-
-          <Column
-            header={
-              <SortableHeaderCell
-                columnContent={<Trans render="span">GPU</Trans>}
-                sortHandler={this.handleSortClick.bind(null, "gpu")}
-                sortDirection={sortColumn === "gpu" ? sortDirection : null}
-              />
-            }
-            cellRenderer={gpuRenderer}
-          />
-
-          <Column
-            header={<span title="Spacing" />}
-            cellRenderer={spacingRenderer}
-            width={spacingSizer}
-          />
-        </Table>
+        <NodesTableWrapper data={data.slice()}>
+          <Table data={data.slice()}>
```

Comment:
You could copy this data once and store it in a local

Reply:
Yes will do